### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: core/classes/Core/TimeAgo.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ public function inWords($past, Language $language): string {
             $timeDifference < (($this->_secondsPerMinute * 89) + 29)
         ) {
             // between 44mins30secs and 1hour29mins29secs
```
Comment (Derkades) [inline core/classes/Core/TimeAgo.php:61]: Between roughly 45 minutes and 90 minutes should say "about 1 hour", not "more than an hour". The previous strings were correct. Same goes for other strings you changed.

File: core/classes/Core/TimeAgo.php [hunk 2]
```diff
@@ -128,7 +128,7 @@ public function inWords($past, Language $language): string {
             )
         ) {
             // between 29days23hours59mins30secs and 59days23hours59mins29secs
-            $key = 'about_1_month';
+            $key = 'more_then_1_hour';
```
Comment (Derkades) [inline core/classes/Core/TimeAgo.php:131]: Accidentally copied `more_then_1_hour` the string?

**Comment by Derkades** [comment]
In my opinion, the previous strings were more accurate.

**Comment by Justman10000** [comment]
Since when does About Etwa mean in German? That's something completely new... Then are the German translations incorrectly

**Comment by Justman10000** [comment]
Have the translations fixed

**Pull request closed without merging**